graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/locking/LockJCRNodeMutationExtension.java

```diff
@@ -124,7 +125,20 @@ public boolean unlock(@GraphQLName("type") @GraphQLDefaultValue(DefaultLockTypeP
     public boolean clearAllLocks() {
         try {
             JCRNodeWrapper nodeToUnlock = nodeMutation.getNode().getNode();
-            if (nodeToUnlock.hasPermission("clearLock")) {
+
+            // custom check for permission added in DX 7.3.5.0:
```

Comment:
I would extract the whole logic to check whether or not the user has the permission (including the fact that the permission exist or not) outside of this method (inside this class if fine).